GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+
+Members of this team are nominated either by the Node.js Technical Steering Committee (TSC) or the Node.js Build WG and follow the guidelines provided on the Collaborator Guidelines of the Node.js Build WG. Note that members of the Node.js Web Team might also recommend people for nomination.
+
+### Node.js Web Standards Team (`@nodejs/web-standards`)
+
+The Node.js Web Standards Team is composed by Node.js Collaborators and External Collaborators that have extensive (or expertisè) on Web Standards, such as Ecma262. The Standards Team is responsible for guiding and serving as point of contacts when either Node.js Collaborators, or the Node.js Technical Steering Committee (TSC) or the Web Team requires assistance or guidance regarding Web Standards.
+
+Members of this team are nominated by the Node.js Technical Steering Committee (TSC). Note that members of the Node.js Web Team might also recommend people for nomination.
+
+### Node.js UX & Design Team (`@nodejs/ux-and-design`)
+
+The Node.js UX & Design Team is composed by Node.js Collaborators and External Collaborators that have extensive (or expertisè) on UX & Design. The UX & Design Team is responsible for guiding and serving as point of contacts when members of the Node.js Web Team require assistance or guidance regarding UX & Design.
+
+Often members of this team will collaborate with providing best-practices and guidelines for the Node.js Website, in matters of UX & Design. Members of this team are also responsible for providing feedback on the Node.js Website, and providing feedback on the Node.js Website's design. (For example, when a discussion raises regarding best-practices regarding topics such as CSS, the UX & Design Team has a say on the matter).
+
+Members of this team are nominated by the Node.js Technical Steering Committee (TSC). Note that members of the Node.js Web Team might also recommend people for nomination.
+
+## The Interoperability of the Node.js Web Team
+
+As seen above, the different teams under the Node.js Web Team umbrella are responsible for having the oversight on different aspects of Node.js's Web-related projects. However, it is important to note that the Node.js Web Team is not a set of siloed teams, but rather a set of teams that work together to achieve the same goal: Providing the best Web Experience for Node.js.
+
+Following this line of thought, the Web Infra Team has technical ownership on aspects that involve knowledge on matters relate to Web Infrastructure (CI/CD, Web Frameworks, Cloud Technologies, Web Security and others); The UX & Design Team has technical ownership on aspects that involve knowledge on matters related to UX & Design (CSS, HTML, Accessibility, UX, Design and others); The Website Team has technical ownership on aspects that involve knowledge on matters related to the Node.js Website (Node.js Website's features, Node.js Website's stability, Node.js Website's security and others); and the Standards Team has technical ownership.
```

Comment:
`and the Standards Team has technical ownership....` seems like an incomplete thought

Reply:
Updated!